virt/virt-4-12-release-notes.adoc

```diff
@@ -46,11 +46,13 @@ The SVVP Certification applies to:
 
 //CNV-18488 SVVP for 4.12
 
+//CNV-16414
+* {VirtProductName} has xref:../virt/logging_events_monitoring/virt-virtualization-alerts.adoc#virt-virtualization-alerts[critical alerts] that inform you when a problem occurs that requires immediate attention. Now, each alert has a corresponding description of the problem, a reason for why the alert is occurring, a troubleshooting process to diagnose the source of the problem, and steps for resolving the alert. This feature was previously introduced as a Technology Preview and is now generally available.
```

Comment:
The link will change for 4.12 because the assembly name is changing. The feature is the runbooks, not the alerts. Here is the [PR](https://github.com/openshift/openshift-docs/pull/53142), so you can update the link.

I suggest something along these lines:

"{VirtProductName} has xref:runbooks to help you troubleshoot issues that trigger alerts. The alerts are displayed on the Virtualization > Overview page of the web console. Each runbook describes the meaning and impact of an alert and provides procedures to diagnose and resolve the issue. This feature etc."

We do not need to mention "critical". The runbooks were TP because we only had the capacity t  document the **critical** alert runbooks. That's why we called the TP feature "critical alerts". For GA, all the runbooks are documented.